ui/src/utils/react-utils.tsx

```diff
@@ -125,9 +125,9 @@ export function useTwoPartEffectWithStatus<TEntity>(
 ): LoaderStatus {
     const [loaderStatus, setLoaderStatus] = React.useState<LoaderStatus>(LoaderStatus.Initialized);
 
-    let cancelled = false;
     useEffect(() => {
         const promise = loadFunc();
+        let cancelled = false;
```

Comment:
Reactin dev-moodin löytämä bugi. Kun dev-moodissa useEffect()it ajetaan aina kahdesti, molemmat ajokerrat näkevät saman `cancelled`in, ja siksi kaikki lataukset peruttiin aina. Tätä funktiota ei vaan ollut kutsuttu mistään.

Reply:
Wanhempaa perua, mutta tämä koko `useTwoPartEffectWithStatus` on mielestäni vähintäänkin nimetty jännästi, sillä mulle jää sellainen vaikutelma että käytännössä cancelability on koko tämän funktion raison d'être, ja se ei näy nimessä ollenkaan. Tosin kokonaisuudessaankin tämä on vähän jännä... Tämä funktio lienee olemassa lähinnä sen takia, että Reactissa ei voi ketjuttaa hookkeja ja koko aiempi ajossa oleva kutsuketju halutaan pystyä canceloimaan, ja sen hoitamiseen saattaisi olla muitakin tapoja